### PR TITLE
LPS-156173 Validate the captcha only when is needed

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-156173

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/captcha/index.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"fieldSets": [
+		{
+			"fields": [
+				{
+					"defaultValue": "true",
+					"label": "captcha",
+					"localizable": false,
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/captcha/index.json:8]: This is the default value so we don't need to add it :)

Comment (victorg1991) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/captcha/index.json:8]: Btw, the captcha fragment, has a captcha configuration, what does it means enabling this?

Comment (lfbesada) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/captcha/index.json:8]: It is just a mark to know that this component is a captcha. IMHO we should evaluate to handle it as typeOptions, see https://liferay.slack.com/archives/C038WMVC5E2/p1655892450430279

Comment (lfbesada) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/captcha/index.json:7]: > I've removed the defaultValue. Thanks for the tip ❤️

**Comment by lfbesada** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/8619

**Pull request closed without merging**